spec/node-spec.js

```diff
@@ -240,6 +240,24 @@ describe('node feature', () => {
       child.stdout.on('data', outDataHandler)
     })
 
+    it('does not start the v8 inspector when --inspect is after a -- argument', (done) => {
+      child = ChildProcess.spawn(remote.process.execPath, [path.join(__dirname, 'fixtures', 'module', 'run-as-node.js'), '--', '--inspect'])
+
+      let output = ''
+      function dataListener (data) {
+        output += data
+      }
+      child.stderr.on('data', dataListener)
+      child.stdout.on('data', dataListener)
+      child.on('exit', () => {
+        if (output.trim().startsWith('Debugger listening on ws://')) {
```

Comment:
Will this fail if `ELECTRON_ENABLE_LOGGING` is set? Perhaps we could make this more permissive by checking `/Debugger listening on/.test(output)`

```suggestion
        if (/Debugger listening on/.test(output)) {
```

Reply:
This is copy pasted from the test above and no, the debugger is one of the first things to run and therefore the first thing to log